patches/0002-patch-vlan-profiles-to-be-non-computed.patch

```diff
@@ -0,0 +1,18 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Venelin <venelin@pulumi.com>
+Date: Fri, 12 Jul 2024 17:27:42 +0100
+Subject: [PATCH] patch vlan profiles to be non-computed
+
+
+diff --git a/internal/provider/resource_meraki_networks_vlan_profiles.go b/internal/provider/resource_meraki_networks_vlan_profiles.go
+index 80dbca9..3cb4b2e 100644
+--- a/internal/provider/resource_meraki_networks_vlan_profiles.go
++++ b/internal/provider/resource_meraki_networks_vlan_profiles.go
+@@ -152,7 +152,6 @@ func (r *NetworksVLANProfilesResource) Schema(_ context.Context, _ resource.Sche
+ 						},
+ 						"name": schema.StringAttribute{
+ 							MarkdownDescription: `Name of the VLAN, string length must be from 1 to 32 characters`,
+-							Computed:            true,
+ 							Optional:            true,
```

Comment:
based on your PR description, I'm confused why we don't need to remove `Optional: true` as well; if `name` has to be set by the user it's not optional right?